File: Week 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int level = 0;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    while (height > level)
    {
        for (int n = height - 1; n > level; n--)
        {
            printf(" ");
        }
        for (int n = 0; n < level + 1; n++)
        {
            printf("#");
        }
        printf("  ");
        for (int n = 0; n < level + 1; n++)
        {
            printf("#");
        }
        printf("\n");
        level++;
    }
}

File: Week 6/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for n in range(height):
    print(" " * (height - n - 1), end="")
    print("#" * (n+1), end="")
    print("  ", end="")
    print("#" * (n+1), end="")
    print()
